tt_metal/hw/inc/mod_div_lib.h

```diff
@@ -42,6 +47,8 @@ inline __attribute__((always_inline)) uint32_t udivsi3_const_divisor(uint32_t n)
     } else if constexpr (d == 94) {
         // fast divide for 94 divisor. Handles Banked L1 address generation for E75
         return fast_udiv_94(n);
+    } else if constexpr (d == 124) {
```

Comment:
how is this change related to this PR?

Reply:
When we have multiple command queues, we need more cores, so we take away from the storage cores.  Given we need two storage cores, we remove 4 banks.